pkg/getter/gitgetter.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright The Helm Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package getter
+
+import (
+	"bytes"
+	"strings"
+
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"helm.sh/helm/v3/internal/fileutil"
+	"helm.sh/helm/v3/pkg/gitutil"
+)
+
+// Assigned here so it can be overridden for testing.
+var gitCloneTo = gitutil.CloneTo
+
+// GITGetter is the default HTTP(/S) backend handler
```

Comment:
This `type` should be `GitGetter` because it is `Git`, not `GIT`

Reply:
ok I will fix it.